### ISSUE TITLE
Group chat support

### ISSUE THREAD

**Issue by navjeetc** [aksonov/react-native-xmpp#2]
Does this support group chat? I got one to one chat working but not sure how to send message to  group chat.

**Comment by aksonov** [comment]
No, group chat is not supported at this moment. It is not difficult to implement, but i'm currently out of time, PR would be welcome.

**Comment by navjeetc** [comment]
I am not a IOS developer, otherwise I would love to help.

**Comment by JonathanWi** [comment]
Interested in this as well! If unable to develop at this time, any chance to quickly explain how you would implement such features?

Cheers!

**Comment by llgoer** [comment]
https://github.com/aksonov/react-native-xmpp/pull/5/commits ,may help you

**Comment by hariks789** [comment]
Hi, I used the above code but group chat is not working. Not able to join the room. I would really appreciate if anyone could help me with this.

**Comment by ficoayyappan** [comment]
Hi, Can you provide group chat for android?

**Comment by ershanky** [comment]
Hi, It would be nice if anyone could help for implementing group chat feature.

**Comment by pewh** [comment]
any roadmap about group chat?

**Comment by aksonov** [comment]
No roadmap, sorry. Feel free to submit PR.

**Comment by andrzejbe** [comment]
Hi @aksonov - please check my PR
